lessons/Early-Math/commutative-property/langs/en-us/pages/commutativity-and-code2.adoc

```diff
@@ -40,7 +40,7 @@ Open the @starter-file{comm-and-assoc}. Complete the exploration to determine if
 
 @fitb{}{}
 
-@n Use @proglang to test your prediction. What did you discover? @fitb{}{}
+@n Use @proglang to test your prediction. What did you discover? @fitb{}{@ifsoln{`max` is commutative.}}
```

Comment:
I also think that asking them to do so much writing is overkill, especially given that our audience is younger and this is literally a yes/no answer anyway